actioncable/lib/action_cable/test_helper.rb

```diff
@@ -0,0 +1,124 @@
+module ActionCable
+  # Provides helper methods for testing Action Cable broadcasting
+  module TestHelper
+    extend ActiveSupport::Concern
+
+    included do
+      def before_setup # :nodoc:
+        server = ActionCable.server
+        test_adapter = ActionCable::SubscriptionAdapter::Test.new(server)
+
+        @old_pubsub_adapter = server.pubsub
+
+        server.instance_variable_set(:@pubsub, test_adapter)
+        super
+      end
+
+      def after_teardown # :nodoc:
+        super
+        ActionCable.server.instance_variable_set(:@pubsub, @old_pubsub_adapter)
+      end
+
+      # Asserts that the number of transmited messages to the channel matches the given number.
+      #
+      #   def test_transmissions
+      #     assert_transmissions 0, 'messages'
+      #     ActionCable.server.broadcast 'messages', { text: 'hello' }
+      #     assert_transmissions 1, 'messages'
+      #     ActionCable.server.broadcast 'messages', { text: 'world' }
+      #     assert_transmissions 2, 'messages'
+      #   end
+      #
+      # If a block is passed, that block should cause the specified number of
+      # messages to be transmited.
+      #
+      #   def test_tramsmissions_again
+      #     assert_transmissions 1, 'messages'
+      #       ActionCable.server.broadcast 'messages', { text: 'hello' }
+      #     end
+      #
+      #     assert_transmissions 2, 'messages'
+      #       ActionCable.server.broadcast 'messages', { text: 'hi' }
+      #       ActionCable.server.broadcast 'messages', { text: 'how are you?' }
+      #     end
+      #   end
+      def assert_transmissions(number, channel)
```

Comment:
My first idea was to use word "messages" (because _message_ is what we really send through the cable). But I took "transmissions" from existing tests.

So, the other candidate is "message". "Broadcast" or "message"?